trivia-game/client/src/components/questioncard.js

```diff
@@ -0,0 +1,37 @@
+import {decode} from 'html-entities';
+import { useState} from "react";
+import Scoreboard from "./scoreboard";
+
+const QuestionCard = (props) => {
+    const [answered, setAnswered] = useState(false)
+    const [score,setScore] = useState(0);
+    const [userAnswer, setUserAnswer] = useState();
+    
+    const handleAnswerOptionClick = (userAnswerParam) => {
+      setUserAnswer(userAnswerParam);
+    if (userAnswerParam === props.question.correct_answer) {
+      props.setScore(props.score + 1);
+      setAnswered(true);
+      console.log(props.score)
+    } else {
+      setAnswered(true);
+      console.log(props.score)
+    }
+  }
+
+    return (
+      <div className="question-section">
+        {/*<div>Question {props.questions}</div>*/}
```

Comment:
Was this going to be a heading for the question number? Is this a future feature? Might be a chance to add some context with a TODO comment:
`{// TODO: Add heading with question number}`